price-oracle/src/lib.rs

```diff
@@ -126,6 +128,15 @@ pub mod pallet {
     #[pallet::getter(fn price_data_thousand)]
     pub type PriceDataThousand<T: Config> = StorageMap<_, Twox64Concat, AssetPairId, BucketQueue, ValueQuery>;
 
+    #[pallet::storage]
+    #[pallet::getter(fn accumulator)]
+    pub type Accumulator<T: Config> = StorageValue<_, BTreeMap<AssetPairId, PriceEntry>, ValueQuery>;
+
+    #[pallet::storage]
+    #[pallet::getter(fn oracle)]
```

Comment:
maybe we can call this getter as `oracles` to follow the convention 
```suggestion
    #[pallet::getter(fn oracles)]
```

Reply:
The convention should be singular for maps (at least where it makes sense, might be different if you store a vec of things in a map entry) and has been at Parity IIRC. Using the getter I get one oracle entry, not several.

Reply:
ah okay, I agree with you. I am working on a pallet right now (nft marketplace) and over there we have plural namings for both maps and getters, despite the fact that we store singular objects and not `vec` or so. But then I might need to do some refactorings on my pallet myself x)